src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -534,6 +535,7 @@
         <xsd:attribute name="local-pk" type="xsd:string" />
         <xsd:attribute name="passphrase" type="xsd:string" />
         <xsd:attribute name="ciphers" type="xsd:string" />
+        <xsd:attribute name="buffer" type="xsd:string" />
```

Comment:
is string correct here or is it because of the true/false **or** regex option?

Reply:
String is correct. When you pass true or false, it is handled correctly